bin/run.sh

```diff
@@ -24,17 +24,13 @@ fi
 slug="$1"
 input_dir="${2%/}"
 output_dir="${3%/}"
-meta_config_json_file="${input_dir}/.meta/config.json"
-representation_file="${output_dir}/representation.txt"
-representation_config="${output_dir}/representation.json"
-mapping_file="${output_dir}/mapping.json"
-
+solution_files="$(jq -c '.files.solution' ${input_dir}/.meta/config.json)"
 # Create the output directory if it doesn't exist
 mkdir -p "${output_dir}"
 
 echo "${slug}: creating representation..."
 
-./bin/representer "${input_dir}" "${meta_config_json_file}" "${representation_file}" "${mapping_file}" "${representation_config}"
+./bin/representer -s $solution_files -d "${input_dir}" -o "${output_dir}"
```

Comment:
```suggestion
./bin/representer -s "${solution_files}" -d "${input_dir}" -o "${output_dir}"
```

Side note: in theory, people could also submit additional files. Does the representer also check those?

Reply:
It does check all files in the config.json file

Reply:
Okay, but people could also create files themselves and submit them